schema/validation.go

```diff
@@ -0,0 +1,53 @@
+//go:generate packr2
+
+package schema
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gobuffalo/packr/v2"
+	"github.com/pkg/errors"
+	"github.com/qri-io/jsonschema"
+)
+
+// newSchemaBox returns a *packer.Box with the schema files from the schema sub-directory
+func newSchemaBox() *packr.Box {
+	return packr.New("github.com/cnabio/cnab-go/schema/schema", "./schema")
+}
+
+// ValidateBundle validates the provided bundle bytes against the applicable CNAB-Spec schema
+func ValidateBundle(bytes []byte) ([]jsonschema.ValError, error) {
+	return Validate("bundle", bytes)
+}
+
+// ValidateClaim validates the provided claim bytes against the applicable CNAB-Spec schema
+func ValidateClaim(bytes []byte) ([]jsonschema.ValError, error) {
+	return Validate("claim", bytes)
+}
+
+// Validate validates the provided bytes against the provided CNAB-Spec schemaType
+func Validate(schemaType string, bytes []byte) ([]jsonschema.ValError, error) {
+	schemaData, err := newSchemaBox().Find(fmt.Sprintf("%s.schema.json", schemaType))
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to read the schema data for type %q", schemaType)
+	}
+
+	rs := &jsonschema.RootSchema{}
+	err = json.Unmarshal(schemaData, rs)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to json.Unmarshal %q schema", schemaType)
+	}
+
+	err = rs.FetchRemoteReferences()
```

Comment:
Again still catching up but is this PR supposed to address airgapped scenarios or was that put off for a follow-up? If not, won't we need to also embed https://github.com/cnabio/cnab-spec/blob/master/schema/definitions.schema.json as well so that we have all of the definitions embedded locally and don't need to fetch anything at runtime?

Reply:
Great point.  I've updated in https://github.com/cnabio/cnab-go/pull/188/commits/3783131f4fde52d874e8ea48bd36ff8871496a17 wherein I used a different json schema library for validation after fighting with the qri-io library used elsewhere to do the same... WDYT?